src/pages/dashboard/index.page.tsx

```diff
@@ -38,8 +42,11 @@ import type { Place } from '@/types/Place';
 import { Values } from '@/types/Values';
 import { WorkflowStatus } from '@/types/WorkflowStatus';
 import { Alert, AlertVariants } from '@/ui/Alert';
-import { Box, parseSpacing } from '@/ui/Box';
+import { Box } from '@/ui/Box';
 import { Dropdown, DropDownVariants } from '@/ui/Dropdown';
+import { Icon } from '@/ui/Icon';
+import { Icons } from '@/ui/Icon';
```

Comment:
You can probably combine the imports?
```suggestion
import { Icon, Icons } from '@/ui/Icon';
```